nyaa/templates/layout.html

```diff
@@ -44,8 +44,174 @@
 		<![endif]-->
 	</head>
 	<body>
+		{% set nyaa_cats = [

```

Comment:
> Even the title column is not needed as, currently, the title of the main categories are merely their name and those of the subcategories are a combination of the name of their respective parent category and their own name with "`-`" in between. It can be constructed on templates, unless this rule is just a coincident and may not be upheld in the future.

Most, yes, but not all. `title` will still be needed.
For example, the title for `Anime / Anime Music Video` (1_1) is `Anime - AMV`.

> How about this?

I like the color, it matches the site's theme.
But maybe the other way around? So the blue color will mark the selected category, since it's bolder, it stands out more.